docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,651 @@
+---
+sidebar_label: jira
+description: Integrates with Jira to manage issues, comments, and track promotion workflows.
+---
+
+# `jira`
+
+:::info Enterprise Feature
+This extension is only available in Akuity Promote, you can use this feature in Kargo v1.6+ offered on [Akuity Platform](https://akuity.io/akuity-platform).
+:::
+
+The `jira` extension provides comprehensive integration with Jira, allowing you to create, update, delete, and search for issues, manage comments, and track promotion workflows. This extension is particularly useful for maintaining traceability between your deployment processes and project management activities.
+
+The extension supports various operations including issue management, comment handling, and status tracking, making it a powerful tool for promotion workflows that require coordination with project management systems.
+
+## Credentials Configuration
+
+All Jira operations require proper authentication credentials stored in a Kubernetes secret.
+
+| Name                     | Type     | Required | Description                                                                                                                |
+| ------------------------ | -------- | -------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `credentials.secretName` | `string` | Y        | Name of the secret containing the Jira credentials.                                                                        |
+| `credentials.namespace`  | `string` | N        | Namespace for the credentials secret. Can be project or cluster secret namespace. If empty, the project namespace is used. |
+
+The referenced secret should contain the following keys:
+
+- `domain`: The domain of your Jira instance or Jira api (e.g., `https://yourcompany.atlassian.net`)
+- `username`: Your Jira username or email
+- `password`: Your Jira API token or password
+
+:::info Content Formatting
+When using `description` or `body` fields, use plain text as Jira does not render markdown. For rich formatting, use the ADF (Atlassian Document Format) alternatives `adfDescription` or `adfBody` instead.
```

Comment:
Up to this point in the doc neither `description` nor `body` fields have been mentioned. I'm actually confused, so readers will be also.

Re: ADF, is there any external resource we can link to that covers it?